### PR TITLE
Astyle 2.04

### PR DESCRIPTION
**Pull request by davydden**
This version comes in hand as the formatting rules were changed in
2.05, whereas there are still opensource software which utilise formatting
as provided by 2.04.

### CODE REVIEW

File: astyle204.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+class Astyle < Formula
+  homepage "http://astyle.sourceforge.net/"
+  url "http://downloads.sourceforge.net/project/astyle/astyle/astyle%202.04/astyle_2.04_macosx.tar.gz"
+  sha1 "2aa956c4521a1163da6a8be741786fd89c1f39a7"
+  head "svn://svn.code.sf.net/p/astyle/code/trunk/AStyle"
+
+  bottle do
```
Comment (DomT4) [inline astyle204.rb]: Remove this block. We'll add a new one on pull.

File: astyle204.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
+class Astyle < Formula
+  homepage "http://astyle.sourceforge.net/"
+  url "http://downloads.sourceforge.net/project/astyle/astyle/astyle%202.04/astyle_2.04_macosx.tar.gz"
```
Comment (DomT4) [inline astyle204.rb]: Should use `https://`

File: astyle204.rb [hunk 3]
```diff
@@ -0,0 +1,32 @@
+class Astyle < Formula
```
Comment (DomT4) [inline astyle204.rb]: Need to make the `class` name here match the formula name.

File: astyle204.rb [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  homepage "http://astyle.sourceforge.net/"
+  url "http://downloads.sourceforge.net/project/astyle/astyle/astyle%202.04/astyle_2.04_macosx.tar.gz"
+  sha1 "2aa956c4521a1163da6a8be741786fd89c1f39a7"
```
Comment (DomT4) [inline astyle204.rb]: Needs to use `sha256`.

File: astyle204.rb [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+  head "svn://svn.code.sf.net/p/astyle/code/trunk/AStyle"
```
Comment (DomT4) [inline astyle204.rb]: Remove the head.

**Comment by DomT4** [comment]
This will either need to be `keg_only` or have a `conflicts_with "astyle"` line added one empty line below the existing bottle block. Otherwise people will be able to link them on top of each other and cause all sorts of trouble.

**Comment by davydden** [comment]
i fixed issues:
```
$ brew audit --strict astyle204.rb
==> brew style astyle204

1 file inspected, no offenses detected
```

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-versions/commit/d2969fe7b053a2c4dde00cec2eb393fbf06922b0. Thank you for your contribution to Versions, we appreciate it! :smiley_cat:.

It's a nit, but the commit message for new formulae should be `astyle204 2.04 (new formula)` or such, Thanks!

**Pull request closed without merging**